src/libmongoc/tests/test-mongoc-sample-commands.c

```diff
@@ -3703,13 +3703,146 @@ _test_sample_versioned_api_example_4 (void)
+                       "quantity", BCON_INT32 (1),
+                       "date", BCON_DATE_TIME (iso_to_unix ("2021-02-03T09:00:00Z")));
+   docs[2] = BCON_NEW ("_id", BCON_INT32 (3),
+                       "item", "xyz",
+                       "price", BCON_INT32 (5),
+                       "quantity", BCON_INT32 (5),
+                       "date", BCON_DATE_TIME (iso_to_unix ("2021-02-03T09:05:00Z")));
+   docs[3] = BCON_NEW ("_id", BCON_INT32 (4),
+                       "item", "abc",
+                       "price", BCON_INT32 (10),
+                       "quantity", BCON_INT32 (10),
+                       "date", BCON_DATE_TIME (iso_to_unix ("2021-02-15T08:00:00Z")));
+   docs[4] = BCON_NEW ("_id", BCON_INT32 (5),
+                       "item", "xyz",
+                       "price", BCON_INT32 (5),
+                       "quantity",BCON_INT32 (10),
+                       "date", BCON_DATE_TIME (iso_to_unix ("2021-02-15T09:05:00Z")));
+   docs[5] = BCON_NEW ("_id", BCON_INT32 (6),
+                       "item", "xyz",
+                       "price", BCON_INT32 (5),
+                       "quantity",BCON_INT32 (5),
+                       "date", BCON_DATE_TIME (iso_to_unix ("2021-02-15T12:05:10Z")));
+   docs[6] = BCON_NEW ("_id", BCON_INT32 (7),
+                       "item", "xyz",
+                       "price", BCON_INT32 (5),
+                       "quantity",BCON_INT32 (10),
+                       "date", BCON_DATE_TIME (iso_to_unix ("2021-02-15T14:12:12Z")));
+   docs[7] = BCON_NEW ("_id", BCON_INT32 (8),
+                       "item", "abc",
+                       "price", BCON_INT32 (10),
+                       "quantity",BCON_INT32 (5),
+                       "date", BCON_DATE_TIME (iso_to_unix ("2021-03-16T20:20:13Z")));
+   ok = mongoc_collection_insert_many (
+      sales, (const bson_t**) docs, N_DOCS, NULL /* opts */, &reply, &error);
+   /* End Versioned API Example 5 */
+   ASSERT_OR_PRINT (ok, error);
+   bson_destroy (&reply);
+
+   cmd = BCON_NEW ("count", "sales");
+   ok = mongoc_database_command_simple (
+      db, cmd, NULL /* read_prefs */, &reply, &error);
+   ASSERT_ERROR_CONTAINS (error,
+                          MONGOC_ERROR_SERVER,
+                          323,
+                          "Provided apiStrict:true, but the command count is not in API Version 1");
+   ASSERT (!ok);
+   bson_destroy (&reply);
+#if 0
+   /* This block not evaluated, but is inserted into documentation to represent the above reply.
+    * Don't delete me! */
+   /* Start Versioned API Example 6 */
+   char *str = bson_as_json (&reply, NULL /* length */);
+   printf ("%s", str);
+   /* Prints the server reply:
+    * { "ok" : 0, "errmsg" : "Provided apiStrict:true, but the command count is not in API Version 1", "code" : 323, "codeName" : "APIStrictError" } */
+   bson_free (str);
+   /* End Versioned API Example 6 */
+#endif
+
+   /* Start Versioned API Example 7 */
+   filter = bson_new ();
```

Comment:
Ok, and looks like this is destroyed later, via bson_destroy(). +1

Reply:
Yup!